cypress/e2e/tests/pages/cluster-manager.spec.ts

```diff
@@ -115,6 +134,25 @@ describe('Cluster Manager', { tags: '@adminUser' }, () => {
         });
       });
 
+      it('can download YAML', () => {
+        // Delete downloads directory. Need a fresh start to avoid conflicting file names
+        cy.deleteDownloadsFolder();
```

Comment:
i'm being paranoid... but to confirm, this just deletes whatever download folder is associated with an isolated cypress browser session?

Reply:
That is correct.